CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to Forgerock's Javascript SDK
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Forgerock and its packages, which are hosted in the [Forgerock](https://github.com/forgerock) on GitHub.
```

Comment:
This may read better like this:

" ... in the [ForgeRock organization](https://github.com/forgerock) on GitHub."

Also, remember that it's always "ForgeRock", not "Forgerock".

Reply:
should it be 'repo' or 'space' instead of 'organization'?